src/sections/Meshery/Features-section/features-section.style.js

```diff
@@ -101,6 +101,11 @@ const FeaturesSectionWrapper = styled.section`
 							max-height: 20rem;
 							margin: auto;
 						}
+
+						StaticImage {
+							max-height: 20rem;
+							margin: auto;
+						}
```

Comment:
I don't think this will actually be applied, because the rendered element is simply an `img` wrapped within a `div`. The `StaticImage` component is just the naming done for it in the gatsby-image plugin.

Reply:
Actually it worked for me. It did fix the styling for me.

Reply:
Yeah, just checked it seems like the CSS for that name is passed down to the `img` tags. Nice work from gatsby.
Though, it would be better to use `.gatsby-image-wrapper img` to assign CSS to images using the `StaticImage` component.

Reply:
Okay 😄

Reply:
Where did we land on this suggestion?